package.json

```diff
@@ -166,12 +166,8 @@
 		"@types/uuid": "^8.3.4",
 		"@types/validator": "^13.7.1",
 		"@types/webpack-env": "^1.16.3",
-		"@types/wordpress__api-fetch": "^3.2.4",
-		"@types/wordpress__block-library": "^2.6.1",
```

Comment:
block-library isn't used in calypso. While the other packages are, they bundle their own types (for quite a while actually)